_includes/pbc/all/install-features/202407.0/install-the-image-search-ai-feature.md

```diff
@@ -0,0 +1,429 @@
+search.with.your-images.content,Téléchargez une image pour trouver des produits similaires,fr_FR
+search.with.your-images.content,Cargue una imagen para encontrar productos similares,es_ES
+search.with.your-images.error,Something went wrong. Please try another image,en_US
+search.with.your-images.error,Etwas ist schiefgelaufen. Bitte versuchen Sie es mit einem anderen Bild,de_DE
+search.with.your-images.error,Une erreur s'est produite. Veuillez essayer avec une autre image,fr_FR
+search.with.your-images.error,"Algo salió mal. Por favor, intente con otra imagen",es_ES
+search.with.your-images.button,Upload an image,en_US
+search.with.your-images.button,Bild hochladen,de_DE
+search.with.your-images.button,Télécharger une image,fr_FR
+search.with.your-images.button,Cargar una imagen,es_ES
+search.with.your.images.photo.button,Take a photo,en_US
+search.with.your.images.photo.button,Foto aufnehmen,de_DE
+search.with.your.images.photo.button,Prendre une photo,fr_FR
+search.with.your.images.photo.button,Hacer una foto,es_ES
+search.with.your-images.no-results,"No results found. Please try another image.",en_US
+search.with.your-images.no-results,"Keine Ergebnisse gefunden. Bitte versuchen Sie es mit einem anderen Bild.",de_DE
+search.with.your-images.no-results,"Aucun résultat trouvé. Veuillez essayer une autre image.",fr_FR
+search.with.your-images.no-results,"No se encontraron resultados. Por favor, intente con otra imagen.",es_ES
+```
+
+2. Import data:
+
+```bash
+console data:import glossary
+```
+
+**Verification**
+
+Make sure that the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables.
+
+### Enable controllers
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN   | SPECIFICATION                                         | PREREQUISITES   | NAMESPACE |
+| -------- |-------------------------------------------------------| --------------- | --------- |
+| ImageSearchAiRouteProviderPlugin | Adds the image search routes to the Yves application. | | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
+
+**src/Pyz/Yves/Router/RouterDependencyProvider.php**
+```php
+    /**
+     * @return array<\Spryker\Yves\RouterExtension\Dependency\Plugin\RouteProviderPluginInterface>
+     */
+    protected function getRouteProvider(): array
+    {
+        return [
+            ...
+            new ImageSearchAiRouteProviderPlugin(),
+        ];
+    }
+```
+
+To apply the changes, run the following command:
+
+```bash
+yves router:cache:warm
+```
+
+**Verification**
+
+Url `http://mysprykershop.com/search-ai/image` will not return `404 Not Found`.
+
+### Set up widget
+Set up the widget as follows:
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```
+    /**
+     * @return array<string>
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            ...
+            ImageSearchAiWidget::class,
+        ];
+    }
```

Comment:
please check how it is done an many other guides, list of plugins goes first, etc